guides/common/modules/ref_capsule-server-scalability-considerations.adoc

```diff
@@ -4,7 +4,7 @@
 = {SmartProxyServer} Scalability Considerations
 
 The maximum number of {SmartProxyServer}s that {ProjectServer} can support has no fixed limit.
-The tested limit is 17 {SmartProxyServer}s with 2 vCPUs on a {ProjectServer} with {RHEL} 8.
+The tested limit is 17 {SmartProxyServer}s with 2 vCPUs on a {ProjectServer}.
```

Comment:
I'm wondering if we should remove this altogether until we re-test it.

Reply:
You could tone it down and say something like "17 proxies w/ 2 CPUs has been tested".